### DISCUSSION TITLE
output to JSON file (for NodeRed)

### DISCUSSION THREAD

**Discussion by maxusaurelius** [kellerza/sunsynk#88] [Q&A] [answered]
hi all,

I'm sorry for this stupid question...
is possible get data from inverter with running script `python3 sunsynk/sunsynk/sunsynk.py` for using with NodeRed (I don't have HA)?
If yes, how...?
if i run this script nothing happens..

**Comment by kellerza** [comment, marked as answer]
Sunsynk is a library, so you can write your own python program to do this

- Import sunsynk
- Configure serial/driver settings
- read
- Write to json
